core/storage/user/gae_models.py

```diff
@@ -1244,12 +1317,17 @@ def apply_deletion_policy(cls, user_id):
         datastore_services.delete_multi(
             cls.query(cls.user_id == user_id).fetch(keys_only=True))
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single shared instance."""
+        return base_models.EXPORT_METHOD.MULTIPLE_UNSHARED_INSTANCES
+
     @classmethod
     def get_export_policy(cls):
         """Model contains user data."""
         return dict(super(cls, cls).get_export_policy(), **{
             'user_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'exploration_id': base_models.EXPORT_POLICY.EXPORTED,
+            'exploration_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
This seems weird, why change it? A lot of the exported values are exploration-specific.

Reply:
The exploration id is used for indexing into the subdictionaries (so it is not exported directly as a key).